cmd/zoekt-webserver/main.go

```diff
@@ -437,25 +434,41 @@ func watchdog(dt time.Duration, maxErrCount int, addr string) {
 	}
 }
 
+func diskUsage(path string) (*disk.UsageStat, error) {
+	duPath := path
+	if runtime.GOOS == "windows" {
+		duPath = filepath.VolumeName(duPath)
+	}
+	usage, err := disk.Usage(duPath)
+	if err != nil {
+		err = fmt.Errorf("diskUsage: %w", err)
```

Comment:
nit: conventional go would just return error here
```suggestion
		return nil, fmt.Errorf("diskUsage: %w", err)
```